substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/c/CAnnotationProcessorCache.java

```diff
@@ -102,6 +109,20 @@ public CAnnotationProcessorCache() {
                 clearCache();
             }
         }
+
+        if (Options.CAPQueryCodeDir.getValue() != null) {
```

Comment:
We can also check if CAPQueryCodeDir is set here with `Options.CAPQueryCodeDir.hasBeenSet()`

Reply:
Done